### PR TITLE
Integration test docker compose

### PR DESCRIPTION
**Pull request by moroten**
This PR upgrades the Buildbarn versions and adds integrations test for the docker-compose deployment.

### CODE REVIEW

**Comment by moroten** [comment]
@EdSchouten Feel free to review these commits, otherwise I'm happy to submit them without your review on Monday.

File: kubernetes/worker-ubuntu16-04.yaml [hunk 1]
```diff
@@ -38,8 +38,8 @@ spec:
           valueFrom:
             fieldRef:
               fieldPath: metadata.name
-      - command: [/bb/tini, -v, -g, --, /bb/bb_runner, /config/runner-ubuntu16-04.jsonnet]
-        image: l.gcr.io/google/rbe-ubuntu16-04@sha256:b516a2d69537cb40a7c6a7d92d0008abb29fba8725243772bdaf2c83f1be2272
+      - command: [/bb/tini, -v, -g, --, /bb/bb_runner, /config/runner-ubuntu22-04.jsonnet]
+        image: ghcr.io/catthehacker/ubuntu:act-22.04@sha256:5f9c35c25db1d51a8ddaae5c0ba8d3c163c5e9a4a6cc97acd409ac7eae239448
```
Comment (EdSchouten) [inline kubernetes/worker-ubuntu16-04.yaml:42]: What's this container image? Who is catthehacker?

**Review by EdSchouten — APPROVED** [review]
Hey! Thanks for picking this up!

As usual, I have very little opinion on what goes on in bb-deployments. So this is fine, I guess? Or it isn't? Feel free to drag others in the loop... or just merge this.

File: kubernetes/worker-ubuntu16-04.yaml (hunk 1 above)
Comment (moroten) [reply kubernetes/worker-ubuntu16-04.yaml:42]: That user is producing the images for https://github.com/nektos/act, used for running GitHub Actions locally. I've never used it before, but it sounds reasonable.

Comment (EdSchouten) [reply kubernetes/worker-ubuntu16-04.yaml:42]: Couldn't we use the official Ubuntu images here?

Comment (moroten) [reply kubernetes/worker-ubuntu16-04.yaml:42]: It was about getting all the tooling, e.g. GCC. Maybe there are better variants, I am more than happy to change to something else.

**Pull request merged**